Profiles/ext-ConditionStatusDate.xml

```diff
@@ -1,51 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'd recommend not removing this extension yet. True, it isn't currently used in cio-Condition, but the StatusDate concept is still part of the functional data set. Keeping the extension makes it easier to readd it in the future.

Reply:
I've decided to remove this extension after all, since the text within the extension is not correct anymore, for instance w.r.t. CreationDateTime.